src/main/java/codeu/controller/SendNotification.java

```diff
@@ -0,0 +1,49 @@
+package codeu.controller;
+import org.json.simple.JSONObject;
+
+import java.io.*;
+
+import java.net.HttpURLConnection;
+import java.net.URL;
+import javax.net.ssl.HttpsURLConnection;
+
+public class SendNotification{
+
+  public void sendMsg(String messageContent) throws IOException
+  {
+    String url = "https://fcm.googleapis.com/fcm/send";
+    String API_ACCESS_KEY="AAAAQ8CY2j0:APA91bHKNT-HNdbxyO-eD671RuQlkZOgMS7VTz66uwtxVr8kDfUDcGCLojY1hgRsXK9IfyE1LmTYarOO6gq_4CThif_6bjgmvh6JIikM28HZTAQs-u7jtEkEUokDHrpWvZ9jsI6Bmfyl";
```

Comment:
Maybe we could keep this in datastore?

Reply:
Keep the function in datastore?

Reply:
Ah, sorry — I meant the secret key. That way, it won't show up in the (public) source code.